src/main/play-2.6/modules/reactivemongo/MongoController.scala

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2012-2013 Stephane Godbillon (@sgodbillon)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package play.modules.reactivemongo
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+import play.api.http.{ HttpChunk, HttpEntity }
+import play.api.mvc.{ BodyParser, MultipartFormData, ResponseHeader, Result }
+
+import reactivemongo.api.bson.{ BSONDocument, BSONValue }
+import reactivemongo.api.bson.collection.BSONSerializationPack
+
+import akka.stream.Materializer
+
+object MongoController {
```

Comment:
There was no need for that, how is it related with support for Play 3?

Reply:
The MongoController Trait references akka dependencies in the signature of functions and needed to be changed to pekko. 
I though that the best way of changing from akka to pekko was to split the trait into a version specific file, resulting in the duplication of the file for every version under play 3.0.